cob_default_env_config/envs/izs-campus/line_maps/lines_top_deck.yaml

```diff
@@ -0,0 +1 @@
+
```

Comment:
remove file? and cob_default_env_config/envs/izs-campus/line_maps/ground_marking.yaml